sdk/lib/opentelemetry/sdk/trace/span.rb

```diff
@@ -347,11 +347,22 @@ def trim_span_attributes(attrs)
           nil
         end
 
+        def truncate(value, limit)
+          case value
```

Comment:
would it make sense to move this logic into `Utilities.truncate` and then potentially dispatch to different methods based on that type if necessary?

Reply:
Hm, yes, most likely. I'll make that change.

Reply:
Updated.